webapp/test_runs_handler.go

```diff
@@ -25,9 +25,9 @@ func testRunsHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func handleTestRunGet(w http.ResponseWriter, r *http.Request) {
-	from, err := shared.ParseFromParam(r)
+	from, err := shared.ParseDateTimeParam(r, "from")
```

Comment:
I feel like we should also support `to` in `/test-runs`. And on a second thought, I think we should just use `ParseTestRunFilterParams` and `LoadTestRuns` (unless there's some reason that I'm not aware of), but that'd be a bigger change and I can send a follow-up PR to clean up this file.

Reply:
Done.